jj.js

```diff
@@ -29,3 +29,34 @@ contactLink.addEventListener('click', () => {
   close();
 })
 
+
+const name1 = document.getElementById('name_1')
+const email = document.getElementById('email_1')
+
+
+
+
+
+const form = document.getElementById('form');
+form.addEventListener('submit', function (event) {
+  // event.preventDefault()
+
+  // getting the values of the inputs;
+  const namevalue = name1.value.trim()
+  const emailvalue = email.value.trim()
+  console.log(namevalue)
+
+  const details = {
+    name: namevalue,
+    email: emailvalue
```

Comment:
- [ ] kindly ensure that you get and save your Textarea input to local storage using this object also and kindly ensure that you wrap the functionality of storing to localStorage  in an input event listener  because it is a project requirement that when the user changes input value, the localStorage should be updated.
- [ ] Your functionality does not work as expected where the user data gets saved to local storage as the user is typing because you're wrapping your code to store user data in a submit function and it won't work until the user submits the form

### EXAMPLE
```
form.addEventListener('input', () => {
  // getting the values of the inputs;
  const namevalue = name1.value.trim()
  const emailvalue = email.value.trim()
  console.log(namevalue)


  const details = {
    name: namevalue,
    email: emailvalue
  }

  localStorage.setItem('details', JSON.stringify(details));
})
```